aion/lib/aion/room_channel/room.ex

```diff
@@ -6,25 +6,30 @@ defmodule Aion.RoomChannel.Room do
   alias Aion.RoomChannel.{Room, UserRecord}
   require Logger
 
+  @type new_question_set :: %{questions: (list Question.t), current_question: Question.t, answers: list Answer.t}
```

Comment:
You're not consequent in the syntax. In `questions` you use parentheses, in `answers` you don't. Pick one, preferably the latter.

Reply:
I can pick the first as from what I experienced, you cannot omit parentheses around list X when it isn't the last key-value pair

Reply:
Fair enough